test/AppLike.jsx

```diff
@@ -1,12 +1,23 @@
 import React from 'react'
 
+import { CozyProvider } from 'cozy-client'
 import { I18n } from 'cozy-ui/transpiled/react/providers/I18n'
 import en from '../src/locales/en.json'
 
-export const AppLike = ({ children }) => {
+const TestI18n = ({ children }) => {
   return (
     <I18n lang="en" dictRequire={() => en}>
       {children}
     </I18n>
   )
 }
+
+const AppLike = ({ children, client }) => {
+  return (
+    <CozyProvider client={client}>
+      <TestI18n>{children}</TestI18n>
+    </CozyProvider>
+  )
+}
+
+export { TestI18n, AppLike }
```

Comment:
What is the goal and benefit? What is the cost to use AppLike everywhere?

Reply:
The aim was to have a lighter wrapper to match with other apps, but I've never done a comparison to find out if there was better performance